packages/react/src/validationSpecKeys.ts

```diff
@@ -0,0 +1,13 @@
+export const RequiredValidationId = "gadget/validation/required";
+export const EmailValidationId = "gadget/validation/email-address";
+export const PasswordValidationId = "gadget/validation/password";
+export const UrlValidationId = "gadget/validation/url";
+export const StringSizeValidationId = "gadget/validation/string-size";
+export const NumberRangeValidationId = "gadget/validation/number-range";
+export const RegexValidationId = "gadget/validation/regexp";
+export const ColorValidationId = "gadget/validation/color";
+export const VectorDimensionsValidationId = "gadget/validation/vector-dimensions";
+export const UniqueValidationId = "gadget/validation/unique";
+export const InvokeCodeValidationId = "gadget/validation/code";
+export const FileSizeRangeValidationId = "gadget/validation/file-size-range";
+export const ValidImageFileValidationId = "gadget/validation/valid-image-file";
```

Comment:
I think it might be a touch simpler to use the GraphQL typename of the object instead of these spec ids. GraphQL forces us to have a typename for every object, and so why have another identifier that just means the same thing here, you know? Will avoid an unnecessary increase in the bundle size as well